plugins/inputs/smart/smart.go

```diff
@@ -282,16 +304,96 @@ func gatherDisk(acc telegraf.Accumulator, usesudo, attributes bool, smartctl, no
 			// save the raw value to a field.
 			if field, ok := deviceFieldIds[attr[1]]; ok {
 				if val, err := parseRawValue(attr[8]); err == nil {
-					device_fields[field] = val
+					deviceFields[field] = val
 				}
 			}
+		} else {
+			var tempC int64
+			if collectAttributes {
+				if startStop := sasStartStopAttr.FindStringSubmatch(line); len(startStop) > 1 {
+					tags["id"] = "4"
+					tags["name"] = "Start_Stop_Count"
+					i, err := strconv.ParseInt(strings.Replace(startStop[1], ",", "", -1), 10, 64)
+					if err != nil {
+						continue
+					}
+					fields["raw_value"] = i
+
+					acc.AddFields("smart_attribute", fields, tags)
```

Comment:
Once we `AddFields`, can we continue to go to the next line without further matching?

Reply:
Probably